tools/phenomenal/ms/ipo/macros.xml

```diff
@@ -21,9 +21,6 @@
         sh -c "exit \$return"
     </token>
 
-    <token name="@COMMAND_NSLAVES@">
-        nSlaves \${GALAXY_SLOTS:-1}
```

Comment:
I wouldn't remove this, I would just set a sensible default after the :- part that makes sense with the assignment of the tool in the resource bins (tiny, small, medium, large, xlarge).

Reply:
Ok so setting `nSlaves \${GALAXY_SLOTS:-4}` works with a destination set to large only, so I think it is only a matter of material, I will put back GALAXY_SLOTS then, thanks for the remark.
I think also that last time I didn't changed the resources allocated to minikube at start (`minikube start --cpus 4 -- memory 8192`), which was certainly the cause of the problem.

Reply:
Great! In the future what we should aim for is to make those slaves processes distributable in Galaxy in some way. Current issue you have is that all these slaves are unfortunately running on the same machine. Maybe we could generate a fake collection of files signalling the different slaves process that we want, and then having another galaxy box to run that part receiving that collection of files plus the R object with the data serialised in a file, and then another box for merging results.

Reply:
We would need to deal somehow with the iterative nature of the process though...

Reply:
That would be awesome indeed, I don't know if galaxy can (already) manage this kind of recursive process